src/HealthChecks.AzureServiceBus/AzureServiceBusTopicHealthCheck.cs

```diff
@@ -36,17 +36,7 @@ public AzureServiceBusTopicHealthCheck(string endpoint, string topicName, TokenC
         {
             try
             {
-                var connectionKey = $"{ConnectionKey}_{_topicName}";

```

Comment:
By the way, it seems a bug was here: `ConnectionKey` is property that already contains `_topicName`.

Reply:
And consider to add `protected override string ConnectionKey => _connectionKey ??= ...` in both health checks to cache string in field though it is not necessary.

Reply:
Done.